README.md

```diff
@@ -21,7 +21,7 @@ to be run from within OctoPrint. The scripts need to define a few magic variable
   successfully, and a dictionary of settings items and their values to persist in the settings
   store.
   
-  Finally your object should have an attribute `response_string`. If set to anything other than
+  Finally, your object should have an attribute `script_status`. If set to anything other than
```

Comment:
This is a breaking change.  How did you decide to handle it?  If we were following semver, the version would get bumped by a major version, or at least a minor since we're still technically on 0.